client/TopBar.jsx

```diff
@@ -1,14 +1,116 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+// import { FormGroup, InputGroup, FormControl } from 'react-bootstrap';
+import styled from 'styled-components';
+
+const Wrapper = styled.section`
+  padding: 4em;
+  background: #ffffff;
+  max-width: 650px;
+  max-height: 200px;
+  padding-top: 0px;
+  padding-left: 20px;
+
+`;
+
+const Title = styled.h2`
+  font-size: 1.5em;
+  color: red;
+  padding-right: 7px;
+`;
+
+const SearchText = styled.textarea`
+  resize: none;
+  height: 24px;
+  width: 200px;
+  padding-right: 0px;
+  font-size: 1em;
+`;
+
+const SearchButton = styled.button`
+  width: 30px;
+  height: 31px;
+  padding-left: 0px;
+  margin: 0px;
+  background-color: red;
+`;
+
+const SearchOptions = styled.form`
+  display: flex;
+  align-items: center;
+`;
+
+const DropDown = styled.select`
+  height: 30px;
+  width: 100px;
+  margin-left: 50px;
+  background-color: #fff;
+  border: 0px;
+  font-size: 1em;
+`;
+
+const EmptyRatingBar = styled.div`
+  display: flex;
+  flex-direction: row;
+  border-radius: 4px;
+  width: 600px;
+  align-items: center;
+  justify-content: center;
+`;
+
+const RatingBarRightSide = styled.span`
+  background-color: #F0F0F0;
+
+display: flex;
+flex-direction: column;
+align-items: center;
+justify-content: center;
+`;
+
+const EmptyUserInfo = styled.span`
+  display:flex;
+  flex-direction: column;
+  width: 300px;
+  align-items: left;
+  padding-left: 10px;
+`;
+
+const StarsBar = styled.span`
+  height: 50px;
+  width: 300px;
+  display:flex;
+  justify-content: center;
+  align-items: center;
+  justify-content: center;
+`;
+
+const BlankPhoto = styled.img`
+  height: 60px;
+  width: 150px;
+  `;
+
+const StartReview = styled.a`
+  padding-bottom: 3px;
+  margin-bottom: 3px;
+  padding-top: 1px;
+`;
 
 export default class TopBar extends React.Component {
   constructor() {
     super();
     this.state = {
-      hovered: 0,
-      grey: [],
-      clicked: false,
+      value: '',
```

Comment:
I usually only use var name 'value' if I am storing a number. Try to use something like searchString so it's more precise.

Reply:
Done. Thanks.